src/pipx/main.py

```diff
@@ -373,7 +374,7 @@ def add_include_dependencies(parser: argparse.ArgumentParser) -> None:
 def add_python_options(parser: argparse.ArgumentParser) -> None:
     parser.add_argument(
         "--python",
-        default=DEFAULT_PYTHON,
+        default="",
```

Comment:
Why even set a default and not let it default to None (which is the default behavior of argparse)?